### ISSUE TITLE
It does not work with multiple scopes

### ISSUE THREAD

**Issue by kozickikarol** [bk201-/jira-prepare-commit-msg#436]
Branch: `feature/TEST-1`
Example commit message:
`feat(scope1,scope2.scope3): new feature`

It changes commit message to:
`[TEST-1] feat(scope1,scope2.scope3): new feature`
which causes that commitlint throws errors:
```
✖   subject may not be empty [subject-empty]
✖   type may not be empty [type-empty]
```

Expected commit message:
`feat(scope1,scope2.scope3): [TEST-1] new feature`

**Comment by benghann** [comment]
Hey @kozickikarol I think this should work https://github.com/bk201-/jira-prepare-commit-msg#conventional-commit . Just update the config.

**Comment by javimsevilla** [comment]
@kozickikarol multiple scopes are not part of the [conventional commits definition](https://www.conventionalcommits.org/). But you can make a fork and change the [conventional commit regexp](https://github.com/bk201-/jira-prepare-commit-msg/blob/540f5c11e41cdd253e9425f20a599bac0f8721cc/src/git.ts#L16) to allow multiple scopes.

**Comment by bk201-** [comment]
Hi! I've fixed it in 1.7.0. Could you please check your cases?
